explainability-service/src/main/java/org/kie/trustyai/service/endpoints/consumer/ConsumerEndpoint.java

```diff
@@ -111,83 +94,9 @@ public Response consumeInput(InferencePartialPayload request) throws DataframeCr
                 return Response.serverError().entity(message).status(Response.Status.BAD_REQUEST).build();
             }
         } else {
-            return Response.serverError().entity("Unsupported payload kind=" + request.getKind()).status(Response.Status.BAD_REQUEST).build();
+            return Response.serverError().entity("Unsupported payload kind=" + request.getKind()).status(Response.Status.INTERNAL_SERVER_ERROR).build();
         }
 
         return Response.ok().build();
     }
-
-    @POST
-    @Consumes(MediaType.APPLICATION_JSON)
-    @Produces(MediaType.TEXT_PLAIN)
-    @Path("/upload")
-    public Response directConsumption(ModelInferJointPayload jointPayload) throws DataframeCreateException {
-        if (jointPayload.getRequest() == null || jointPayload.getRequest().getInputs().length < 1) {
```

Comment:
just a confirmation that this is getting removed because the `/upload` method has already been moved to `DataEndpoint`.

Reply:
Yep, I meant to remove it from the /consume endpoint in the last PR